Source/VolunteerReporting/Web.Angular/src/app/casereport/casereportlist/casereportlist.component.html

```diff
@@ -0,0 +1,32 @@
+<p>
+    case report list works!
+</p>
+
+<table>
+    <thead>
+        <tr>
+            <th>Time</th>
+            <th>Data Collector</th>
+            <th>Health Risk</th>
+            <th>Females &gt; 5</th>
```

Comment:
To avoid breaking the heading, you may use no-breaking space instead of breaking space. So something like this:

`<th>Females&nbsp;&gt;&nbsp;5</th>`

Which avoids:

![image](https://user-images.githubusercontent.com/1311807/31792827-9f8be57e-b51d-11e7-87a7-e53d2a33e9fc.png)

Reply:
Like this table will live for long, lol :) But, thanks, it's fixed